commit 538cd0175e38e49ca5a8d6ddab4fd0a20bfda9d1 (refs/changes/10/4710/2)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-08-17T00:42:10+00:00 (1 year, 2 months ago)
    
    OS-7142 mbstowcs(3C) is the wrong man page
    Reviewed by: Jason King <jbk@joyent.com>
    Approved by: Jason King <jbk@joyent.com>

diff --git a/manifest b/manifest
index 587a1000d1..9c02a48c02 100644
--- a/manifest
+++ b/manifest
@@ -13417,8 +13417,8 @@ s usr/share/man/man3c/mbsnrtowcs.3c=mbsrtowcs.3c
 s usr/share/man/man3c/mbsnrtowcs_l.3c=mbsrtowcs.3c
 f usr/share/man/man3c/mbsrtowcs.3c 0444 root bin
 s usr/share/man/man3c/mbsrtowcs_l.3c=mbsrtowcs.3c
-s usr/share/man/man3c/mbstowcs.3c=mbsrtowcs.3c
-s usr/share/man/man3c/mbstowcs_l.3c=mbsrtowcs.3c
+f usr/share/man/man3c/mbstowcs.3c 0444 root bin
+s usr/share/man/man3c/mbstowcs_l.3c=mbstowcs.3c
 f usr/share/man/man3c/mbtowc.3c 0444 root bin
 s usr/share/man/man3c/mbtowc_l.3c=mbtowc.3c
 s usr/share/man/man3c/memalign.3c=malloc.3c
